packages/utils/src/index.ts

```diff
@@ -122,3 +122,4 @@ export type {
   WithRequired,
 } from './lib/types.js';
 export { verboseUtils } from './lib/verbose-utils.js';
+export { coreConfigMessageBuilder } from './lib/zod-validation.js';
```

Comment:
Since the function is applicable to any Zod schema, not just core config, a name such as `zodErrorMessageBuilder` would be more appropriate.